packages/helix-shared-config/test/specs/sitemapconfigs/multilang.json

```diff
@@ -13,6 +13,12 @@
           "destination": "/sitemap-fr.xml",
           "hreflang": "fr",
           "alternate": "/fr/{path}"
+        },
+        "la": {
+          "source": "/la/query-index.json",
+          "destination": "/sitemap-la.xml",
+          "hreflangs": ["es-VE","es-SV","es-PA"],
+          "alternate": "/la/{path}"
```

Comment:
this is the actual use case to solve: one `<loc>` for every entry in `la` but producing multiple alternates in various hreflangs